integration-tests/jpa-mysqldb/pom.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
maybe rename the directory of the integration tests from `jpa-mysqldb` to `jpa-mysql` ?